docs/source/css/qb1-sphinx-rtd.css

```diff
@@ -475,6 +475,4 @@ footer span.commit .rst-content tt,
     display: block;
     content: ' ';
     width: 1px;
-    height: 60px;
-    margin-top: -80px;
```

Comment:
As per Richard's comment though:

> The best way to fix this is to make both height and margin-top to have the same magnitude, e.g. 60px and -60px, or 80px and -80px, so that they cancel each other out.